Gemfile.lock

```diff
@@ -405,69 +408,70 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
```

Comment:
### [maxitest](https://github.com/grosser/maxitest)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.3.0 → 3.7.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/maxitest/3.3.0/3.7.0)_]





<details>
<summary>Commits</summary>

A change of **24** commits. See the full changes on [the compare page](https://github.com/grosser/maxitest/compare/v3.3.0...v3.7.0).

These are the first 10 commits:
- (5964309) [test on 5.12](https://github.com/grosser/maxitest/commit/5964309ca6295eea34652a85f76f8f01a5e9720d)
- (5235d20) [512 is broken on old rubies](https://github.com/grosser/maxitest/commit/5235d2087fe64d2d1f5d3674a11007e7ba9a4567)
- (f9b7e85) [Merge pull request #29 from grosser/grosser/12](https://github.com/grosser/maxitest/commit/f9b7e8595bf98184c382b625ab1332adf1f3c2b6)
- (bc1817b) [v3.4.0](https://github.com/grosser/maxitest/commit/bc1817b35e5a75b20f23ab21be04986180af4bd3)
- (2ffb7be) [move todo into issues](https://github.com/grosser/maxitest/commit/2ffb7be31a5cf9238ce3ffd18e5a1b82af9565e7)
- (9d1b2a2) [support 5.12.12](https://github.com/grosser/maxitest/commit/9d1b2a267294b22fc7b68ba19bf8ed5aa68c67c0)
- (e3a3755) [Merge pull request #31 from grosser/grosser/5121](https://github.com/grosser/maxitest/commit/e3a3755835783e9b21e52c332ad3c54fde6e8cc3)
- (ffaa03a) [next minitest supports ruby 2.2](https://github.com/grosser/maxitest/commit/ffaa03a2582e478947509af68770eed35080c7e0)
- (f886477) [Merge pull request #32 from grosser/grosser/5122](https://github.com/grosser/maxitest/commit/f8864773783c3926525c478f905e37995a3175fd)
- (cc6f258) [513](https://github.com/grosser/maxitest/commit/cc6f25840ccaedbd226f19dd9db378d612ad606c)


</details>